etc/profile-a-l/exfalso.profile

```diff
@@ -4,58 +4,12 @@
 # Persistent local customizations
 include exfalso.local
 # Persistent global definitions
-include globals.local
+# added by included profile
+#include globals.local
 
-noblacklist ${HOME}/.quodlibet
-noblacklist ${MUSIC}
-
-# Allow python (blacklisted by disable-interpreters.inc)
-include allow-python2.inc
-include allow-python3.inc
-
-whitelist ${DOWNLOADS}
-whitelist ${MUSIC}
-
-include disable-common.inc
-include disable-devel.inc
-include disable-exec.inc
-include disable-interpreters.inc
-include disable-passwdmgr.inc
-include disable-programs.inc
-include disable-shell.inc
-include disable-xdg.inc
-
-mkdir ${HOME}/.quodlibet
-whitelist ${HOME}/.quodlibet
-include whitelist-common.inc
-include whitelist-usr-share-common.inc
-include whitelist-var-common.inc
-
-caps.drop all
-ipc-namespace
-machine-id
-netfilter
-no3d
-nodvd
-nogroups
-nonewprivs
-noroot
-nosound
-notv
-nou2f
-novideo
-protocol unix,inet,inet6
-seccomp
-shell none
-
-private-bin exfalso,python*
-private-cache
-private-dev
-private-etc alternatives,fonts,group,passwd
```

Comment:
Hi,
for private-etc, I quickly tested exfalso without any apparent problem, hence my modification. If I need to, I can add the problem line.
Is there anything else to expect because I would like it to be the last PR :)
Thanks for the answer.